libkbfs/tlf_handle.go

```diff
@@ -204,6 +198,52 @@ func (h TlfHandle) Extensions() (extensions []TlfHandleExtension) {
 	return extensions
 }
 
+// Equals returns whether h and other contain the same info.
+func (h TlfHandle) Equals(codec Codec, other TlfHandle) (bool, error) {
```

Comment:
A little worried about how brittle this is, although this is probably an issue with several of our data types.

A random idea: what do you think about putting a line of code like this in here (or something similar outside of a function in library initialization, so it doesn't get executed every time):
```
// TODO: If you ever get this failure, you need to update this function to account for the new field.
if reflect.ValueOf(h).NumField() != 8 {
    return false, "Unexpected number of fields in TlfHandle"
}
```